estate/views/estate_menus.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8" ?>
+
+<odoo>
+
+    <menuitem id="estate_property_menu_root" name="Estate Property Menu">
+        <menuitem id="estate_property_level_menu" name="First Level">
+            <menuitem id="estate_property_menu_action" action="estate_property_menu_action"/>
```

Comment:
This menuitem has the same id as the action. Do not mention `menu` in the action id, this will solve the data import issue you are encountering.